### ISSUE TITLE
start() is being called twice on components spawned during the first frame

### ISSUE THREAD

**Issue by pqftgs** [UPBGE/blender#87]
Sample blend http://www.pasteall.org/blend/41491
Build hash 7e45369

This component spawns an object:
```
import bge


class Spawn(bge.types.KX_PythonComponent):
    def start(self, args):
        self.addCylinder()
        
    def addCylinder(self):
        print ("Spawning a cylinder...")
        object = self.object
        scene = object.scene
        ob = scene.addObject('Cylinder', object)

    def update(self):
        return
```

This one is attached to the spawned object and prints whenever start() is called:
```
import bge


class Comp(bge.types.KX_PythonComponent):
    def start(self, args):
        print ("Calling start()")
        print (self.object, "children: ", self.object.children)
        
    def update(self):
        return
```

Console output, which all seems to occur before "Blender Game Engine Started"
```
Spawning a cylinder...
Calling start()
Cylinder children:  []
Calling start()
Cylinder children:  [childcube]

Blender Game Engine Started
Blender Game Engine Finished
```
The problem is that start() is being called twice for one object.  (Possibly related: object.children isn't populated being populated on the first call, as you can see from the output.)

**Comment by panzergame** [comment]
@pqftgs : I see an other issue that your. start is called twice but for the original cylinder object (in the hiden layer) and the duplicated cylinder, normal, no ?

The bug i see is that the duplicated object at component initialization cannot get to its children, this is fixed in : 560efc5.

**Comment by pqftgs** [comment]
Thanks.  Wasn't sure if I should have made a separate bug report.

**Comment by panzergame** [comment]
Fixed in 3a4ccdf. thanks for report.
